src/app/recipes/recipe-edit/recipe-edit.component.ts

```diff
@@ -72,21 +88,17 @@ export class RecipeEditComponent implements OnInit, OnDestroy {
 
   private initForm() {
     let recipeName = '';
-    let recipeImagePath = '';
     let recipeDescription = '';
+    let image = null;
     let recipeIngredients = new FormArray([]);
 
     if (this.editMode) {
       this.recipeSub = this.store
-        .pipe(
-          select('recipes'),
-          map((recipeState) => {
-            return recipeState.recipes.find((recipe) => recipe._id === this.id);
-          })
-        )
+        .pipe(select(RecipesSelectors.findRecipeById, { id: this.id }))
         .subscribe((recipe) => {
+          this.imagePreview = environment.DataBaseUrl + recipe.imagePath;
+          this.EditImage = true;
```

Comment:
Please combine rxjs operators instead `subscribe`. Here you can use `map` operator after `select` in `pipe`. In map you will toggle `EditImage` flag and return receipt, that you will store in class variable. In template you can use pipe `async` to show data

Reply:
fixed